src/js/main.js

```diff
@@ -130,6 +130,35 @@ function styleInputsToObject(inputs) {
 function getDatatemplate(value) {
   return '[data-template="' + value + '"]';
 }
+
+function initModal() {
+  // Get the modal.
+  var modal = document.getElementById('terms-modal');
+
+  // Get the button that opens the modal.
+  var btn = document.getElementById('terms-btn');
+
+  // Get the <span> element that closes the modal.
+  var span = document.getElementsByClassName('close')[0];
+
+  // When the user clicks the button, open the modal.
+  btn.onclick = function () {
+    modal.style.display = 'block';
+  }
+
+  // When the user clicks on <span> (x), close the modal.
+  span.onclick = function () {
+    modal.style.display = 'none';
+  }
+
+  // When the user clicks anywhere outside of the modal, close it.
+  window.onclick = function (event) {
+    if (event.target === modal) {
```

Comment:
- I guess here should be `event.target !== modal` if we want to close the modal when clicking outside of it, and also `event.target` should be checked for parent element as well, because when clicking on any other element inside this modal e.g. <span> it'll close the modal because event.target in this case will be span element not modal.